walkthroughs/3-low-code-api-development/walkthrough.adoc

```diff
@@ -347,19 +347,17 @@ By default, Fuse Online does not expose an *Integration* to the public internet
 === Find the Greeting Service Internal Hostname
 
 . Login to the link:{openshift-console-url}[OpenShift Console, window="_blank"].
-. Select the project named with the following format: `fuse-<UNIQUE_ID>`.
-. Choose *Applications > Services* from the left menu.
-. Select *i-greeting-integration-{user-username}* from the *Services* list.
-. Note the *Hostname* listed on the *Service Details* screen. It will have a format similar to `i-greeting-integration-{user-username}.fuse-<UNIQUE_ID>.svc.cluster.local`
+. Select the project named with the following format: `redhat-rhmi-fuse`.
```

Comment:
Users will use the shared fuse in OS4?

Reply:
Good question. No fuse instance got provisioned in the WT namespace, is that expected?